build.gradle

```diff
@@ -29,6 +29,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:3.4.1'
```

Comment:
Please add link to relevant docs for performance monitoring to README.md. Just a one line mention with a link should be sufficient.

Reply:
Done